pyaptamer/utils/__init__.py

```diff
@@ -1,9 +1,14 @@
 """Utils for the pyaptamer package."""
```

Comment:
Just observed this, can you move everything out of utils into their respective files? It breaks the purpose of having util files for separate purposes if we move some functions to root import imo

Reply:
This is something we had discussed in the weekly regarding the structure of utils, the main essence of the discussion was that init should only contain imports which are going to be used by users and algorithms for transformations, etc. Not private functions that serves the use case of a single algorithm, in that sense we should also move literally every function in utils to init, hence resulting in no structure for the file.